server/metadata/tenant.go

```diff
@@ -112,10 +112,13 @@ type TenantManager struct {
 }
 
 func NewTenantManager() *TenantManager {
+	idToTenantMap := make(map[uint32]string)
+	idToTenantMap[DefaultNamespaceId] = DefaultNamespaceName
```

Comment:
is it causing some test failure? Otherwise, the reload method on the namespace manager should fill it during service start.

Reply:
It did break a test and I put a workaround in place to expect this default. I only added this because the collection name lookup only worked if the tenant was created in the same instantiation.